finalPrep/README.md

```diff
@@ -18,37 +18,40 @@ Some things to consider as you refactor include...
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+<img width="1056" alt="Screen Shot 2021-08-07 at 9 11 08 AM" src="https://user-images.githubusercontent.com/81824035/128605017-767ec68b-29d7-4177-8069-0c220082f137.png">
 - `Add Week 2 Screenshot Here`
+<img width="1058" alt="Screen Shot 2021-08-07 at 9 11 35 AM" src="https://user-images.githubusercontent.com/81824035/128605024-af80458e-6e0f-4e5f-9645-e12ed670e2a4.png">
 - `Add Week 3 Screenshot Here`
+<img width="1036" alt="Screen Shot 2021-08-07 at 9 11 48 AM" src="https://user-images.githubusercontent.com/81824035/128605026-80a6664a-14bd-41ed-9a99-5914ad3da11b.png">
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/Echory/3a64a7e7042cda6f55a5ba833c73d494
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: https://gist.github.com/Echory/fbec2f60b9458f29b5fc785857bd9c7e
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: https://gist.github.com/Echory/31f909f414d45974effc6b2a19ac646c
```

Comment:
Nice self-awareness here- you will develop your opinions on these matters during the course of Turing. A few things to highlight:
- I am like you and prefer verbal communication. However, communication through software like slack and github will be expectations of any job you get, so be sure to also challenge yourself to grow in this area.
- I'd say a good place to start with giving feedback is to try to model how you like to receive feedback. That being said, it sounds like you are very open to communication with your groups, so hopefully this is something that can be discussed openly.